qa-tests-backend/src/test/groovy/ProcessesListeningOnPortsTest.groovy

```diff
@@ -52,6 +53,15 @@ class ProcessesListeningOnPortsTest extends BaseSpecification {
                     .setExposeAsService(true)
                     .setCommand(["/bin/sh", "-c",])
                     .setArgs(["(socat "+SOCAT_DEBUG+" TCP-LISTEN:8081,fork STDOUT)" as String,]),
+            new Deployment()
+                    .setName(TCPCONNECTIONTARGET3)
+                    .setImage("quay.io/rhacs-eng/qa:socat")
+                    .addPort(8082, "TCP")
+                    .addLabel("app", TCPCONNECTIONTARGET3)
+                    .setExposeAsService(true)
+                    .setCommand(["/bin/sh", "-c",])
+                    .setArgs(["(socat "+SOCAT_DEBUG+" TCP-LISTEN:8082,fork STDOUT & " +
+			"sleep 90 && pkill socat && sleep 3600)" as String,]),
```

Comment:
A listening endpoint is created and 90 seconds later the listening endpoint is destroyed.